sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -309,6 +309,41 @@ You should see output similar to:
     node.js version     :  v6.10.3
     npm version         :  3.10.10
 
+The next check that can be made is to generate one of the example figures.
+There are a examples for the different ways in which bokeh can be used.
+
+To create an html file,
+
+.. code-block:: sh
+
+    env BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
+
+which will create a file ``iris.html`` locally and open up a web browser.
+
+.. image:: /_images/bokeh_iris_html.png
+    :scale: 50 %
+    :align: center
+
+The variable ``BOKEH_RESOURCES`` determines where the css and JavaScript
+resources required by bokeh are found. In this case they are included inline in
+the html file. Another possible value is ``BOKEH_RESOURCES=cdn`` which will fetch the resources
+from http://cdn.pydata.org Other options can be found in the
+`resources reference`_.
```

Comment:
CDN is the default so I don't think this last sentence is necessary. Also if you are trying to test locally build BokehJS using CDN resources is always what you *don't* want to do, so we could state why `BOKEH_RESOURCES` is needed instead.